vosekast_control/TestSequence.py

```diff
@@ -190,7 +190,7 @@ def pause_sequence(self):
 
         # set fill countdown to False
         for tank in self.vosekast.tanks:
-            tank.fill_state = False
+            tank.state = "BETWEEN"
```

Comment:
Is 'BETWEEN' still needed if you have `IS_FILLING`, `IS_DRAINING` and `STOPPED`.

Reply:
will be fixed in next commit